### PR TITLE
[DOCS][KSPM] Benchmark rules - new doc

### LINKED ISSUE #2274: [DOCS][CSPM][8.4] Create Benchmark rules page
## Description 

Create a page in the Cloud Native Security section called "Benchmark rules" that describes the Benchmark rules page. It should link to the KSPM integration page, and possibly to the CS Dashboard page as well. 

## Acceptance Test Criteria

Approved by Tinsae and docs team

## Notes

meta issue: #2260

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2274 by creating a Benchmark rules page.

Preview: [Benchmark rules](https://security-docs_2332.docs-preview.app.elstc.co/guide/en/security/master/benchmark-rules.html)

Meta: 
#2260

### CODE REVIEW

File: docs/cloud-native-security/benchmark-rules.asciidoc [hunk 1]
```diff
@@ -1,2 +1,7 @@
 [[benchmark-rules]]
 = Benchmark rules
+The Benchmark Integrations page helps you view and manage the CSP benchmark rules that are active for each of your <<kspm, KSPM integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of <<kspm, Findings>>.
```
Comment (joepeeples) [inline docs/cloud-native-security/benchmark-rules.asciidoc:3]: It'd probably be good to spell out CSP and KSPM on these first references. Also the reference to "Findings" seems a little obscure, esp if this is the first time learning about it -- maybe a little more context:
```suggestion
The Benchmark Integrations page helps you view and manage the cloud security posture (CSP) benchmark rules that are active for each of your <<kspm, Kubernetes security posture management (KSPM) integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of the results on the <<kspm, Findings page>>.
```

File: docs/cloud-native-security/benchmark-rules.asciidoc [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+
+To view the benchmark rules associated with a policy, click that policy's name. You can then click on a benchmark rule's name to see details, including information about how to remediate failures, and related links.
+
+You can't edit benchmark rules, but you can enable and disable them for each agent integration policy. They default to `Enabled`. You can enable or disable them individually, or in bulk by using the *Bulk Actions* menu.
```
Comment (joepeeples) [inline docs/cloud-native-security/benchmark-rules.asciidoc:7]:
```suggestion
You can't edit benchmark rules, but you can enable and disable them for each agent integration policy. They default to `Enabled`. You can enable or disable them individually, or in bulk with the *Bulk Actions* menu. 
```

**Review by joepeeples — COMMENTED** [review]
Solid content -- maybe add a general screenshot, if possible?

File: docs/cloud-native-security/benchmark-rules.asciidoc [hunk 3]
```diff
@@ -1,2 +1,7 @@
 [[benchmark-rules]]
 = Benchmark rules
+The Benchmark Integrations page helps you view and manage the cloud security posture (CSP) benchmark rules that are active for each of your <<kspm, Kubernetes security posture management (KSPM) integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of the results on the <<kspm, Findings page>>.
+
+To view the benchmark rules associated with a policy, click that policy's name. You can then click on a benchmark rule's name to see details, including information about how to remediate failures, and related links.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/benchmark-rules.asciidoc:5]:
```suggestion
To view the benchmark rules associated with a policy, click that policy's name. You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
```

File: docs/cloud-native-security/benchmark-rules.asciidoc [hunk 4]
```diff
@@ -1,2 +1,7 @@
...
+
+You can't edit benchmark rules, but you can enable and disable them for each agent integration policy. They default to `Enabled`. You can enable or disable them individually, or in bulk with the *Bulk Actions* menu.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/benchmark-rules.asciidoc:7]:
```suggestion
You can't edit benchmark rules, but you can enable and disable them for each {agent} integration policy. They default to `Enabled`. You can enable or disable them individually, or in bulk with the *Bulk Actions* menu. 
```

**Review by nastasha-solomon — COMMENTED** [review]
Just two minor edits. :) Looks great!

File: docs/cloud-native-security/benchmark-rules.asciidoc [hunk 5]
```diff
@@ -1,2 +1,7 @@
 [[benchmark-rules]]
 = Benchmark rules
+The Benchmark Integrations page helps you view and manage the cloud security posture (CSP) benchmark rules that are active for each of your <<kspm, Kubernetes security posture management (KSPM) integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of the results on the <<kspm, Findings page>>.
+
+To view the benchmark rules associated with a policy, click that policy's name. You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
+
+You can't edit benchmark rules, but you can enable and disable them for each {agent} integration policy. They default to `Enabled`. You can enable or disable them individually, or in bulk with the *Bulk Actions* menu.
```
Comment (jmikell821) [inline docs/cloud-native-security/benchmark-rules.asciidoc:7]:
```suggestion
You can enable or disable benchmark rules for each integration policy, either individually or in bulk using the *Bulk Actions* menu. By default, benchmark rules are enabled. 
```

File: docs/cloud-native-security/benchmark-rules.asciidoc [hunk 6]
```diff
@@ -1,2 +1,7 @@
 [[benchmark-rules]]
 = Benchmark rules
+The Benchmark Integrations page helps you view and manage the cloud security posture (CSP) benchmark rules that are active for each of your <<kspm, Kubernetes security posture management (KSPM) integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of the results on the <<kspm, Findings page>>.
+
+To view the benchmark rules associated with a policy, click that policy's name. You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
```
Comment (jmikell821) [inline docs/cloud-native-security/benchmark-rules.asciidoc:5]: Suggestion for a better flow. 
```suggestion
To view the benchmark rules associated with a policy, click that policy's name. You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links. 

NOTE: Benchmark rules are not editable. 
```

**Review by jmikell821 — COMMENTED** [review]
Just a few suggestions ;)

**Pull request merged**